docs/devops/getting-started.md

```diff
@@ -12,25 +12,28 @@ keywords:
 - cloud native
 - cloud sdk
 - sap cloud sdk
+- jenkins
+- continuous integration
+- continuous delivery
+- project "piper"
 image:
 ---
 
 ## Best DevOps practices at SAP ##
-Continuous integration and delivery are pillars for contemporary application development. DevOps is the first class
-citizen in SAP Cloud SDK and shipped as best practice with all generated project templates.
 
-### Continuous delivery toolkit ###
+Continuous delivery is a method to develop software with short feedback cycles.
+It is applicable to projects both for SAP Cloud Platform and SAP on-premise platforms.
+SAP implements tooling for continuous delivery in project "Piper" and SAP Cloud SDK for Continuous Delivery.
+The goal of project "Piper" is to substantially ease setting up continuous delivery in your project using SAP technologies.
 
-SAP Cloud SDK for Pipeline is Open Source.
+### SAP Cloud SDK for Continuous Delivery ###
 
-[This repository](https://github.com/SAP/cloud-s4-sdk-pipeline) contains a Jenkins Pipeline-as-Code that enables
-development teams to assure a high level of quality on the process as well as code level. While the SAP Cloud SDK enables us to
-build high-quality cloud applications, this pipeline helps to assure high quality on the project level by a set of standard
-and SDK-specific quality checks.
+SAP Cloud SDK for Continuous Delivery is Open Source and easy to use.
 
-### Project Piper ###
+[The GitHub repository](https://github.com/SAP/cloud-s4-sdk-pipeline) contains a Jenkins Pipeline-as-Code that enables development teams to assure a high level of quality on the process as well as code level.
+While the SAP Cloud SDK enables us to build high-quality cloud applications, this pipeline helps to assure high quality on the project level by a set of standard and SDK-specific quality checks.
 
-In great park pipeline and action configuration for Cloud SDK is based on open-sourced SAP library dubbed [Project
-Piper](https://sap.github.io/jenkins-library/guidedtour/ ).
+### Project "Piper" ###
 
-[Please, check the official documentation for more details.](https://github.com/SAP/cloud-s4-sdk-pipeline)
+For more information on CI/CD in the SAP ecosytem, please refer to the documentation of [project "Piper"](https://sap.github.io/jenkins-library).
+It also holds the [documentation of SAP Cloud SDK for Continuous Delivery](https://sap.github.io/jenkins-library/pipelines/cloud-sdk/introduction/).
```

Comment:
This should be more prominent, as it is - besides the introduction - the main message of this page, isn't it?